src/room/GroupCallView.tsx

```diff
@@ -219,7 +220,9 @@ export function GroupCallView({
     undefined
   );
 
-  const livekitServiceURL = groupCall.foci[0]?.livekitServiceUrl;
+  const livekitServiceURL =
+    groupCall.foci[0]?.livekitServiceUrl ??
+    Config.get().livekit.livekit_service_url;
```

Comment:
I guess I am assuming correctly, that a failed network request or an unset value will result in foci[0] to be undefined?

Reply:
That's actually the following step. This only falls back when the value is undefined (i.e. the `m.call` event doesn't have the field). After we get the URL here we pass it over to the `OpenIDLoader` which actually does the request